.github/workflows/checks.yaml

```diff
@@ -19,7 +19,10 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@v3.0.0
         with:
-          go-version: 1.17
+          go-version: 1.18
+
+      - name: install kubectl-kcp
+        run: curl -L https://github.com/kcp-dev/kcp/releases/download/v0.8.2/kubectl-kcp-plugin_0.8.2_linux_amd64.tar.gz | tar -xz -C $HOME/go
```

Comment:
Any particular reason `$HOME/go` is used? Maybe `/usr/local/bin` is a better place?

Reply:
I thought plugins had to go there. I updated it to `/usr/local/bin`

Reply:
Just somewhere in `$PATH`, I think.